library/libosbuild/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "libosbuild"
+version = "73.0.0"
```

Comment:
Quite an ambitious version :smile:

Reply:
Hah, yea osbuild itself doesn't follow semver so for now (at least initially) I've put it to the same version as osbuild itself.